content/Alliance/_index.md

```diff
@@ -5,12 +5,10 @@ weight = 2
 pre = "<b>2. </b>"
 +++
 
-
-# The Connectivity Standards Alliance
+# The Connectivity Standards Alliance (The Alliance)
```

Comment:
```suggestion
# The Connectivity Standards Alliance
```
I agree that its useful to explain acroynms but this makes the H1 on the page far to long

Reply:
Done